source/_integrations/workday.markdown

```diff
@@ -151,7 +151,7 @@ binary_sensor:
 Example usage for automation:
 
 ```yaml
-automation:
+automation workday_heating:
```

Comment:
This is not a generic approach. Yes, that works, but it is using an advanced configuration method most people should not need.

Lets revert this change.